src/components/GradientPicker/index.js

```diff
@@ -44,12 +49,15 @@ const GradientPicker = ({
 	children,
 	flatStyle = false,
 	onPaletteChange,
-	onColorStopSelect = noop
+	onColorStopSelect = noop,
+	direction = DEFAULT_DIRECTION,
+	popoverColorPicker = false
 }) => {
 	palette = mapIdToPalette(palette);
-
+	const [isPickerOpen, setPickerOpen] = React.useState(false);
```

Comment:
Assume u take my suggesting for the naming, this should have default by it's mode
```suggestion
	const [showColorPicker, setShowColorPicker] = React.useState(colorPickerMode === 'static');
```